css/styles.css

```diff
@@ -0,0 +1,121 @@
+body  {
+  background-color: black;
+  border-bottom: 5px solid #222222;
+  border-top: 5px solid #222222;
+  padding: 50px;
+
+}
+
+h1  {
+  font-family: sans-serif;
+  text-align: center;
+  text-transform: uppercase;
+  font-weight: bold;
+  text-align: center;
+  color: #FFFFFF;
+  margin-top: 0;
+}
+
+hr  {
+  border: none;
+  border-top: 1px solid #222222;
+  margin: 30px 0;
+}
+
+h2  {
+  color: #FFFFFF;
+}
+
+h3  {
+  color: #FFFFFF;
+  font-size: 30px;
+  font-family: sans-serif;
+  font-weight: normal;
+}
+
+p {
+  color: #FFFFFF;
+  font-family: sans-serif;
+  padding-left: 5px;
+
+}
+
+a:link {
+    color: #ffffff;
+    background-color: transparent;
+    font-weight: 500;
+    font-family: sans-serif;
+    text-decoration: none;
+}
+
+a:hover {
+  color: #8FBC8F;
+
+}
+
+img.head  {
+  border-top: 20px;
```

Comment:
@Schultisal you could have used a short cut and written `padding: 20px` since all sides are the same value.

Reply:
@rmjames Thank you for the reply and comments. I appreciate it the feedback! I see what you mean too about the padding; after looking back at my code.